src/lib/pages/schedule-view.svelte

```diff
@@ -53,8 +53,8 @@
 
   import type { DescribeScheduleResponse } from '$types';
 
-  let namespace = $page.params.namespace;
-  let scheduleId = $page.params.schedule;
+  $: namespace = $page.params.namespace;
+  $: scheduleId = $page.params.schedule;
```

Comment:
This breaks the page since the below let scheduleFetch = fetchSchedule(parameters) is called with namespace/scheduleId as undefined.

Reply:
You can test this by creating a schedule and then trying to go to it